.github/workflows/dotnet-core.yml

```diff
@@ -0,0 +1,55 @@
+name: .NET Core
+
+on:
+  push:
+    branches: [ main, release* ]
+  pull_request:
+    branches: [ main, release* ]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Required for GitVersion
+          submodules: 'recursive'
+      
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.101
+
+      - name: Install GitVersion        
+        uses: gittools/actions/gitversion/setup@v0.9.9
+        with:
+          versionSpec: '5.6.6'
+
+      - name: Show GitVersion Config
+        id: gitversion
+        uses: gittools/actions/gitversion/execute@v0.9.9
+        with:
+          useConfigFile: true
+          additionalArguments: '/showConfig'
+
+      - name: Determine Version and Update csproj Files
+        uses: gittools/actions/gitversion/execute@v0.9.9
+        with:
+          useConfigFile: true
+          additionalArguments: '/updateprojectfiles'
+    
+      - name: Display updated csproj File
+        run: cat Freshli.Cli/Freshli.Cli.csproj
+
+      - name: Build
+        run: dotnet build --configuration Release
+                
+      - name: Package
+        if: github.event_name == 'push'
+        run: dotnet pack -c Release
```

Comment:
The CLI does not produce a NuGet package.  The "Package" and "Publish Alpha Package to GitHub Packages" can be removed.